internal/oci/runtime_vm.go

```diff
@@ -812,31 +803,21 @@ func metricsToCtrStats(ctx context.Context, c *Container, m *cgroups.Metrics) *C
 				m.Memory.TotalInactiveFile, memUsage,
 			)
 		}
-
-		for _, entry := range m.Blkio.IoServiceBytesRecursive {
-			switch strings.ToLower(entry.Op) {
-			case "read":
-				blockInput += entry.Value
-			case "write":
-				blockOutput += entry.Value
-			}
-		}
+		rssBytes = m.Memory.RSS
+		pageFaults = m.Memory.PgFault
+		majorPageFaults = m.Memory.PgMajFault
```

Comment:
For kata-containers, this is the part that differs from the approach taken before and that we should ensure the CRI runtime has access to the correct information.

Those are collect, and then passed up to the CRI runtime, here:
https://github.com/kata-containers/kata-containers/blob/e89c06e68bcd465cc79dc68945751a706f230bfe/src/runtime/pkg/containerd-shim-v2/metrics.go#L98-L146

It means that, theoretically, from the kata-containers everything needed here is already provided by kata-containers, but I still need to get to do some more tests on this.